book/src/1.welcome/README.md

```diff
@@ -1,6 +1,47 @@
 # 📖 Teams AI Library v2
```

Comment:
NIT: Consider centering this?
<center><h1>Teams AI Library v2</h1></center>

Reply:
Is that how you are doing it elsewhere?

Reply:
Does this work in .md?

Reply:
It works in .md

Reply:
> Is that how you are doing it elsewhere?

Tried it out last night and it works

Reply:
Sweet, updating.